src/main/java/com/rehab/model/Admin.java

```diff
@@ -0,0 +1,11 @@
+package com.rehab.model;
+
+public class Admin extends AbstractEmployee {
+
+    public Admin() {
```

Comment:
For example when I initialize:
    Admin admin = new Admin();
What will be when:
   admin.getEmail()
   admin.getPassword()
   admin.getRole()
?

Reply:
null will be returned. I created constructor with no args, because it is supposed to be for JPA.